nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -436,9 +436,6 @@ static void phNciNfc_ResetNtfDelayCb(uint32_t dwTimerId, void *pContext)
     if (NULL != pNciContext)
     {
         (void)phOsalNfc_Timer_Stop(pNciContext->dwNtfTimerId);
-        (void)phOsalNfc_Timer_Delete(pNciContext->dwNtfTimerId);
-
-        pNciContext->dwNtfTimerId = 0;
```

Comment:
I see.
Having this info in commit description is handy but not necessary. I'd rather add a check if dwNtfTimerId is valid before pNciContext->dwNtfTimerId = phOsalNfc_Timer_Create(); line. Something like:
...
if (pNciContext->dwNtfTimerId != PH_OSALNFC_TIMER_ID_INVALID)
{
    [log error]
}
pNciContext->dwNtfTimerId = phOsalNfc_Timer_Create();
...
This would be safer and more informative for a reader